### PR TITLE
Setup Grunt tasks

### PR DESCRIPTION
**Pull request by jakech**
@louisbuchbinder @Ivnmza @liminator1106 

Read the "Setup Local" section on README.md

### CODE REVIEW

**Comment by louisbuchbinder** [comment]
hey jake why did you delete these in the grunt file
-  // grunt.loadNpmTasks('grunt-contrib-uglify');
  -  grunt.loadNpmTasks('grunt-contrib-jshint');
  -  grunt.loadNpmTasks('grunt-contrib-watch');
  -  grunt.loadNpmTasks('grunt-contrib-concat');
  -  grunt.loadNpmTasks('grunt-mocha-test');
  -  // grunt.loadNpmTasks('grunt-shell');
  -  grunt.loadNpmTasks('grunt-nodemon');

**Comment by louisbuchbinder** [comment]
otherwise looks good. :+1:

**Comment by jakech** [comment]
@louisbuchbinder 
there is a grunt plugin called `load-grunt-tasks`

see `line 2` require that and it will auto load all `grunt` tasks from `node_modules`

File: Gruntfile.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 module.exports = function(grunt) {
+  require('load-grunt-tasks')(grunt);
```
Comment (jakech) [inline Gruntfile.js:2]: @louisbuchbinder here

**Comment by louisbuchbinder** [comment]
ok cool I'll check it out

**Pull request merged**